cypress/utils/utils.ts

```diff
@@ -825,6 +826,29 @@ export function createMultipleStakeholders(
     return stakeholdersList;
 }
 
+export function createMultipleMigrationWaves(
+    numberOfMigrationWaves: number,
+    StakeholdersList?: Array<Stakeholders>,
+    stakeholderGroupsList?: Array<Stakeholdergroups>
+): Array<MigrationWave> {
+    let MigrationWaveList: Array<MigrationWave> = [];
+    for (let i = 0; i < numberOfMigrationWaves; i++) {
+        let startDate = new Date("2023-06-28");
```

Comment:
Dates are hard coded until we create a helper method

Reply:
This code will stop working on 2023-06-28 because a Migration Wave must have a start date greater than the actual one.

while we don't have the helper method, you can still use a dynamic date:

```
const now = new Date();
now.setDate(now.getDate() + 1);

const end = new Date(now.getTime());
end.setFullYear(end.getFullYear() + 1);

```